ftw/simplelayout/aliasblock/browser/viewlet.py

```diff
@@ -0,0 +1,23 @@
+from ftw.simplelayout.aliasblock.contents.interfaces import IAliasBlock
+from plone import api
+from plone.app.layout.viewlets.common import ViewletBase
+from Products.Five.browser.pagetemplatefile import ViewPageTemplateFile
+from zope.component import queryMultiAdapter
+
+
+def geo_settings_installed():
+    version = api.portal.get().portal_setup.getLastVersionForProfile('collective.geo.settings:default')
+    return version != 'unknown'
+
+
+
+class AliasBlockFormViewlet(ViewletBase):
+    index = ViewPageTemplateFile('templates/aliasblock_form_viewlet.pt')
+
+    def update(self):
+        super(AliasBlockFormViewlet, self).update()
+
+        is_aliasblock = IAliasBlock.providedBy(self.context)
```

Comment:
Is it not possible to use `for="ftw.simplelayout.contenttypes.contents.interfaces.IAliasBlock"` in the ZCML?

Reply:
@djowett-ftw It would be, but only on the edit_form, the add form is called on the container, therefore on the simplelayout page. This way only 1 viewlet needs to be registered instead of twice almost the same.

Reply:
ok, yeah, makes sense. thanks!